pkgs/native_assets_cli/lib/src/model/target.dart

```diff
@@ -38,6 +38,7 @@ class Architecture {
     Abi.androidArm64: Architecture.arm64,
     Abi.androidIA32: Architecture.ia32,
     Abi.androidX64: Architecture.x64,
+    Abi.androidRiscv64: Architecture.riscv64,
```

Comment:
This file needs to be it's own PR + a version  bump to 0.3.1 so that it can be published before the native_toolchain_c PR.